src/xbps/index.md

```diff
@@ -31,3 +31,13 @@ together with the [xbps.d(5)](https://man.voidlinux.org/xbps.d.5) man page.
 To learn how to build packages from source, refer to [the README for the
 void-packages
 repository](https://github.com/void-linux/void-packages/blob/master/README.md).
+
+## Verifying RSA keys
+
+When installing Void for the first time, you may get a message from xbps claiming:
+
+```
+<REPO> repository has been RSA signed by <RSA-FINGERPRINT>
+```
+
+To verify the signature, ensure the <RSA-FINGERPRINT> matches one of the fingerprints in [Void Linux's repo keys](https://github.com/void-linux/void-packages/tree/master/common/repo-keys).
```

Comment:
You can list them from here as well: https://github.com/void-linux/void-mklive/tree/master/keys . That way we have two sources for them. We might get a tweet for it as well.